source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionIsolationMutex.cs

```diff
@@ -32,25 +32,25 @@ public async Task ScriptIsolationMutexFull_EnsuresTwoDifferentScriptsDontRunAtTh
 
             var secondScriptStart = Path.Combine(clientTentacle.TemporaryDirectory.DirectoryPath, "secondScriptStartFile");
 
-            var firstStartScriptCommand = new LatestStartScriptCommandBuilder()
-                .WithScriptBody(new ScriptBuilder()
+            var firstStartScriptCommand = new TestExecuteShellScriptCommandBuilder()
+                .SetScriptBody(new ScriptBuilder()
```

Comment:
Hey @APErebus,

I know this is a bit late to the party, but why did we change the convention of using "With" methods in our builders for "Set"?

Reply:
So I took a bit of inspiration from the Nuke builders and how `Set` feels more specific than `With`. That being said, if this drastically moves away from our standards (is there any doco on this?), I can change them back

Reply:
Sure, Let me get through these other changes and I'll revert them to `With`